server/src/migration/1555064588740-ManageSessionEntity.ts

```diff
@@ -0,0 +1,35 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class ManageSessionEntity1555064588740 implements MigrationInterface {
+
```

Comment:
Please, check this migration on dev server before merge your branch to master. You could notify me for manual deploy.

Reply:
this migration was generated automatically and i checked that, it worked

Reply:
I understand, but your local env could be different with DBs on dev/qa env. Anyway, I would recommend you to check this migration before merge.

Reply:
ok, i'll do, but i don't know how i can do that. can you help?

Reply:
Yep, is this PR finished or still in progress?

Reply:
finished now

Reply:
@kovnick need your help!